### DISCUSSION TITLE
无法登陆，且提示 That's not even a token

### DISCUSSION THREAD

**Discussion by lovelykids** [AlistGo/alist#2600] [General]
https://thumbsnap.com/i/HKqB8LFq.png?1206
新安装的alist 使用的docker 最新版

退出登陆后，或者点访客访问后，就会提示 That's not even a token

并且无法访问主页，打开：5244后就到登录页面

需要用浏览器隐身访问，才可以正常登录/访问主页

http://home.liuzelin.com:5244/

admin
pRelFhdE

**Comment by lovelykids** [comment]
找到问题，使用内网ip时访问出现此问题

**Comment by ParseDark** [comment]
I have the same issue. But I fix it by change the install path. Hope it can help somebody.
